src/main/java/org/wildfly/security/auth/client/ElytronXmlParser.java

```diff
@@ -774,13 +774,15 @@ static void parseAuthenticationConfigurationType(ConfigurationXMLStreamReader re
                         foundBits = setBit(foundBits, 0);
                         final String hostName = parseNameType(reader);
                         configuration = andThenOp(configuration, parentConfig -> parentConfig.useHost(hostName));
+                        xmlLog.xmlDeprecatedElement(reader.getLocalName(), reader.getLocation());
```

Comment:
I did initially think the same as well but no I think as this is only a deprecation warning and not a disabling of the feature we should warn users at the first available opportunity they are using a deprecated feature.

There is no guarantee that a user would even keep their schemas up to date with our later versions as they upgrade the WildFly Elytron version - for some users it will actually be in their interest to remain on an older version of the schema to retain compatibility with more Elytron versions.